community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -29,10 +29,10 @@
  */
 public class BuiltInProcedures
 {
-    public static void addTo( KernelAPI kernel ) throws ProcedureException
+    public static void addTo( Procedures procs ) throws ProcedureException
     {
-        kernel.registerProcedure( new ListLabelsProcedure( procedureName( "sys", "db", "labels" ) ) );
-        kernel.registerProcedure( new ListPropertyKeysProcedure( procedureName( "sys", "db", "propertyKeys" ) ) );
-        kernel.registerProcedure( new ListRelationshipTypesProcedure( procedureName( "sys", "db", "relationshipTypes" ) ) );
+        procs.register( new ListLabelsProcedure( procedureName( "sys", "db", "labels" ) ) );
```

Comment:
I would assume that predefine procedures should have single predefined name, so there should not be any reason to pass it as a constructor param.
Plus probably name and namespace should be different parameters no need to mix them in varargs.